### DISCUSSION TITLE
Simple starts Question for newbie

### DISCUSSION THREAD

**Discussion by vamoruso** [ludwig-ai/ludwig#3920] [General]
Hello i'm trying to start a train experiment on windows 11
using python 3.10 
when executing the following command
_ludwig train -c train_conf.yaml --dataset final_train.csv  --output_directory .    --data_format csv  --experiment_name train --verbose_
 nothing happen.
I only see in console this message :
_PyTorch version 2.1.2 available._

This is the content of yaml file :
input_features:
  - name: Title
    type: text
  - name: Description
    type: text

output_features:
  - name: ClassIndex
    type: category

trainer:
    # learning_rate: 0.0001
    # decay: True
    # decay_rate: 0.9
    # decay_steps: 30000
    # staircase: True
    epochs: 20

Version of ludwig i 0.9.0 dev.
Thank you in advance.
Vince

**Comment by arnavgarg1** [comment]
Hi @vamoruso! 

Are you able to try this command instead?

```bash
ludwig train -c train_conf.yaml --dataset final_train.csv --output_directory . --data_format csv --experiment_name train --logging_level info
```

We don't support the `verbose` argument; instead, we call it `logging_level`. There are 5 options (but INFO is the one that's recommend if you're not actively developing/contributing to Ludwig): "critical", "error", "warning", "info", "debug".

Let me know if that resolves your issue!

**Reply by vamoruso** [reply]
Hi @arnavgarg1 
tryied info, critical and debug without success.
All i see in console is only "PyTorch version 2.1.2 available."

(venv) D:\Universita\Dispense\Tesi_Materiale>ludwig train -c train_conf.yaml --dataset final_train.csv --output_directory . --data_format csv --experiment_name train --logging_level debug
PyTorch version 2.1.2 available.

(venv) D:\Universita\Dispense\Tesi_Materiale>

Thanks in advance

**Reply by arnavgarg1** [reply]
I see @vamoruso. I tried this on mac and Linux and I can see that it logs to stdout. Unfortunately, I don't have a windows machine so will need to check with the other maintainers and see if anyone can reproduce and debug the issue.

**Reply by vamoruso** [reply]
Thanks a lot, let me know.

**Reply by vamoruso** [reply]
I've tried on a friend's MacOS and it works. @arnavgarg1  is there a suggested Linux distro for Ludwig?.
Thank you for job and support.
Ciao
Vincenzo

**Reply by arnavgarg1** [reply]
Hi @vamoruso! Glad to know you were able to confirm that it works on a Mac. 

Ludwig actually ships with Docker images - would that work for you? I believe it uses the official Torch and Ray docker images, both of which use Ubuntu.

**Reply by arnavgarg1** [reply]
You can check out our various images here: https://hub.docker.com/u/ludwigai

Here's a description of the packages: https://ludwig.ai/latest/getting_started/docker/
